index.md

```diff
@@ -15,63 +17,9 @@ The table found below is now generated from the "source" of the Matroska specifi
 Note that versions 1, 2 and 3 have been finalized. Version 4 is currently work in progress. There may be further additions to v4.
 
 
-### EBML principle
```

Comment:
The EBML section is almost completely removed and a reference to Matroska's dependency on the separate EBML specification is added.